### DISCUSSION TITLE
There is way to increase context length for pretrained models?

### DISCUSSION THREAD

**Discussion by PARSA-MHMDI** [mlfoundations/open_clip#858] [Q&A]
Hi,

I want to use pre-trained models to write image captions. But the contex length is limited to 77 for coca models. Is there a way to increase this amount or are there other models that can be used for this purpose? I want the length of the output text to be more than 77 tokens.

**Comment by BIGBALLON** [comment]
same issue here: https://github.com/mlfoundations/open_clip/issues/857
